internal/information/build_info.go

```diff
@@ -0,0 +1,72 @@
+package information
+
+import (
+	"runtime/debug"
+
+	"github.com/golang/glog"
+)
+
+type BuildInfo struct {
+	commitSha       string
+	architecture    string
+	vcsTime         string
+	vcsType         string
+	operatingSystem string
+	goVersion       string
+}
+
+// GetBuildInfo returns the current Build Information as a BuildInfo type (struct) using a built-in debug package and its ReadBuildInfo() function.
+// Upon error, an empty struct is returned along with a logged error.
+func GetBuildInfo() *BuildInfo {
+	b := new(BuildInfo)
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
+		glog.Fatal("unable to get build info")
+	}
+	b.goVersion = info.GoVersion
+	for _, i := range info.Settings {
```

Comment:
This is correct but could be more efficient.

`info.Settings` is a map. There is no need to iterate among all keys.
We should change the implementation to retrieve the values using the keys instead.